bin/rustdoc

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+USAGE="Builds Tokio's RustDocs.
+
+USAGE:
+    "$0" [FLAGS] [<rustdoc-options>...]
+
+FLAGS:
+    -h, --help      Show this help text and exit
+    -u, --unstable  Build documentation for unstable features
+
+NOTES:
+    Additional arguments are passed to the RustDoc invocation. For
+    example:
+
+        "$0" --open
+
+    will open the documentatiion in a web browser.
+
+    The '-u' or '--unstable' flag will build documentation for unstable
+    features. Unstable features will also be documented if the RUSTFLAGS
+    environment variable contains \"--cfg tokio_unstable\". For example:
+
+        RUSTFLAGS=\"--cfg tokio_unstable\" "$0""
+
+set -e
+
+RUSTFLAGS="${RUSTFLAGS:-}"
+ARGS=""
+while (( $# )); do
+    case "$1" in
+        -h|--help)
+        echo "$USAGE"
+        exit 0
+        ;;
+        -u|--unstable)
+        RUSTFLAGS="--cfg tokio_unstable ${RUSTFLAGS}"
+        ;;
+        *)
+        ARGS="${ARGS} $1"
+        ;;
+    esac
+    shift
+done
+
+RUSTDOCFLAGS="${RUSTDOCFLAGS:-} --cfg docsrs ${RUSTFLAGS}" \
```

Comment:
Why did you merge RUSTFLAGS into RUSTDOCFLAGS? (There are some flags that exist in RUSTFLAGS but not in RUSTDOCFLAGS.)

Reply:
it was a lazy way of adding the unstable cfg if it's set, [bf7454b](https://github.com/tokio-rs/tokio/pull/4544/commits/bf7454b0dc91f7ef3b10bb687610f2ad435dfeb3) changes this to do it the correct way.